extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -77,12 +91,25 @@ export class AnchorAdStrategy {
     if (!optInStatus) {
       return false;
     }
-    for (let i = 0; i < optInStatus.length; i++) {
-      if (optInStatus[i] == OPT_IN_STATUS_ANCHOR_ADS) {
-        return true;
-      }
+    return optInStatus.some(
+      status =>
+        status == OptInStatus.OPT_IN_STATUS_ANCHOR_ADS ||
+        status == OptInStatus.OPT_IN_STATUS_NO_FILL_ANCHOR_ADS
+    );
+  }
+
+  /**
+   * @return {boolean}
+   * @private
+   */
+  hasOptInStatusNoFillAnchorAds_() {
+    const optInStatus = this.configObj_['optInStatus'];
+    if (!optInStatus) {
+      return false;
     }
-    return false;
+    return optInStatus.some(
```

Comment:
Can you just use Array.includes?
The whole function could probably be simplified to:
return (this.configObj_['optInStatus'] || []).includes(OptInStatus.OPT_IN_STATUS_NO_FILL_ANCHOR_ADS);

Reply:
Done.